lib/chef/provider/chef_organization.rb

```diff
@@ -52,7 +52,9 @@ def whyrun_supported?
     new_resource.members.each do |user|
       if !existing_members.include?(user)
         converge_by "Add #{user} to organization #{new_resource.name}" do
-          rest.post("#{rest.root_url}/organizations/#{new_resource.name}/users/#{user}", {})
```

Comment:
Hmmmm.  I'm pretty sure @marcparadise added an API to do exactly this; though it might have been changed after I wrote this resource.  @marcparadise , is there an API that lets you directly add someone to an organization without having to do the association_requests dance?